com.discordapp.Discord.json

```diff
@@ -20,7 +20,6 @@
         "--filesystem=xdg-videos:ro",
         "--filesystem=xdg-pictures:ro",
         "--filesystem=xdg-download",
-        "--env=XDG_CURRENT_DESKTOP=Unity",
```

Comment:
Are you sure this isn't needed?

libappindicator depending on the platform falls back to XEmbed tray icons, which is not what we generally want. This forced it to always use StatusNotifiers.

It may happen to work where you tested it just because of your environment.

Reply:
I don't use a system tray or the dash-to-panel so I have absolutely no way to test this. I was thinking of spinning up a Ubuntu VM to test the libappindicator stuff if no one offered to test soon.

Also, I think I probably misunderstood what you said in a previous review as not needing the Unity env variable when instead it was about `com.canonical.indicator.application` or `com.canonical.AppMenu.Registrar`, not entirely sure...

Reply:
Also worth noting an old version of some Node appindicator library used to check `$XDG_CURRENT_DESKTOP=Unity` but stopped a few releases ago, I'm not sure if this was related to that too?

Reply:
@refi64 Yes that is probably the check I was thinking of. I wonder what logic it has now to figure it out? (I wouldn't be surprised if it was not ideal still)

Reply:
Now the flatpak uses `ubuntu:GNOME` which I would assume doesn't offer XEmbed at least. I think I'll have to contact Discord to figure out what check they do before using `com.canonical.Unity.LauncherEntry` because it's not in upstream electron afaict and potentially how they deal with the appindicator on KDE which allows you to open the program with a left-click although maybe that's possible to find. I could add XEmbed tray icon checking to that list. Also probably should ask if we can be told of updates before they get pushed with a silent download change a day before it gets pushed to clients maybe or something.

Anyhow, can you check if the latest build offers XEmbed tray icons or do you want to open an issue? I'd like to merge this soon.